libs/components/lookup/src/lib/modules/lookup/lookup.component.spec.ts

```diff
@@ -3641,13 +3645,15 @@ describe('Lookup component', function () {
           fixture.detectChanges();
 
           const inputElement = getInputElement(lookupComponent);
+          inputElement.focus();
```

Comment:
Because we used to use the event target - we did not need to actually focus - but now we use `activeElement` and so we do.